content/post/thriving-oss-community.md

```diff
@@ -0,0 +1,138 @@
++++
+title = "How to Build a Thriving Open-source Community"
+draft = false
+date = 2017-11-20T00:00:00Z
```

Comment:
I thought the post was going up on the 21st (or the 28th), but this says the 20th. Is this a placeholder?

Reply:
I was going to put it up tonight and chuck was going to submit to HN tomorrow. Might change to 28th.